src/rpcblockchain.cpp

```diff
@@ -939,7 +941,10 @@ UniValue rainbymagnitude(const UniValue& params, bool fHelp)
             CBitcoinAddress address(sGRCAddress);
 
             if (!address.IsValid())
```

Comment:
Minor nitpick: we can move this above `dTotalAmount += dPayout;` for an edge case that will probably never happen. Otherwise, looks good 👍

Reply:
I agree. It could cause the balance check to fail, although the expectation is that the entire amount requested to be paid out should be in the balance, not the shortpay, so that is debatable.